app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1125,7 +1153,11 @@ class BrowserTabViewModel @Inject constructor(
     ) {
         Timber.v("Page changed: $url")
         hasCtaBeenShownForCurrentPage.set(false)
-        buildSiteFactory(url, title)
+        var stillExternal = false
+        if (urlUnchangedForExternalLaunchPurposes(site?.url, url)) {
+            stillExternal = true
+        }
+        buildSiteFactory(url, title, stillExternal)
```

Comment:
You can simplify this:
```
val stillExternal = urlUnchangedForExternalLaunchPurposes(site?.url, url)
buildSiteFactory(url, title, stillExternal)
```
or 

`buildSiteFactory(url, title, urlUnchangedForExternalLaunchPurposes(site?.url, url))`